.github/workflows/testing-develop.yml

```diff
@@ -4,10 +4,13 @@ on:
   push:
     branches:
     - develop
+    - 'GSOC**'
 
   pull_request:
     branches:
     - develop
+    - 'GSOC**'
```

Comment:
there is a branch name "develop" below...?

Reply:
ah, yes, tried a new fixated solution, because branch dependent we have the env defined.